src/wmtk/simplex/link_condition.cpp

```diff
@@ -0,0 +1,90 @@
+#include "link_condition.hpp"
+#include "link.hpp"
+#include "open_star.hpp"
+
+namespace wmtk::simplex {
+bool link_condition_closed_trimesh(const TriMesh& mesh, const Tuple& edge)
+{
+    SimplexCollection lnk_a = link(mesh, Simplex(PrimitiveType::Vertex, edge), true); // link(a)
```

Comment:
* `const`
* Just write `link_a` instead of `lnk_a`.
* You don't need to explicitly write `true` here. The default is already true.
* If you write something like `const Simplex v_a = Simplex(PrimitiveType::Vertex, edge)` you can shorten down this line by a lot and make it much more readable. The same holds for `v_b` and `e`.